.github/workflows/testnet.yml

```
@@ -0,0 +1,233 @@
+        run: |
+          if [[ "${{ github.event.inputs.network_type }}" == "single" ]]; then
+            docker compose up -d
+          else
+            docker compose -f docker compose.multi.yml up -d
+          fi
+          
+          echo "Waiting for services to start..."
+          sleep 30
+      
+      - name: Monitor testnet health
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC1073:error:1:18: Couldn't parse this $((..)) expression. Fix to allow more checks

(shellcheck)